sdk/core/src/main/java/com/klaviyo/coresdk/Klaviyo.kt

```diff
@@ -41,10 +41,15 @@ object Klaviyo {
      * @param email Email address for active user
      * @return
      */
-    fun setEmail(email: String): Klaviyo = this.setProfile(Profile().setEmail(email))
+    fun setEmail(email: String): Klaviyo = this.setProfileAttribute(ProfileKey.EMAIL, email)
```

Comment:
Maybe I should do this too!

Reply:
You know what they say, use your own apis

Reply:
I was waffling on even having the identifiers as keys. I think I kind of like the idea of recording every identifier that is set back the server. Maybe unlikely but I think if I do it this we might lose repeated setting of the same identifier type within the denounce interval.

Reply:
Yeah this is kind of a side-effect of how `Profile` model is implemented as just a managed `map` object, so everything needed to be keyed.